src/inputs/getDesiredName.ts

```diff
@@ -0,0 +1,22 @@
+"use strict";
+
+import * as vscode from "vscode";
+
+import { CancelError, NoNameGivenError } from "../errors";
+
+export async function getDesiredName(): Promise<string> {
+  const name = await vscode.window.showInputBox({
+    ignoreFocusOut: true,
+    placeHolder: "Name",
+    value: "",
+  });
```

Comment:
Something not really related to the purpose of this PR but it looks like [inputBoxOptions](https://code.visualstudio.com/docs/extensionAPI/vscode-api#_a-nameinputboxoptionsaspan-classcodeitem-id528inputboxoptionsspan) has a `validateInput` method that we could use to prevent the user from submitting no name.  Could add something along the lines of:

`  const name = await vscode.window.showInputBox({
    ignoreFocusOut: true,
    placeHolder: "Name",
    value: "",
    validateInput: (value: string) => !value.trim() ? 'Name must not be empty' : ''
  });`

I can submit a seperate PR if you think this is a good idea?

Reply:
👍👍👍👍👍